Packs/PagerDuty/Integrations/PagerDuty/PagerDuty_test.py

```diff
@@ -428,3 +429,16 @@ def test_translate_severity(mocker, severity, expected_result):
     from PagerDuty import translate_severity
     res = translate_severity(severity)
     assert res == expected_result
+
+
+def test_pagination_incidents_with_page_and_page_size(requests_mock: MockerCore) -> None:
```

Comment:
paging function is rather complex, 
I expect to have something like 4-5 tests with doc sting that explains every case in order to cover it correctly